packages/eslint-plugin-pf-codemods/test/rules/v5/divider-remove-isVertical.js

```diff
@@ -0,0 +1,24 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/divider-remove-isVertical');
+
+ruleTester.run("divider-remove-isVertical", rule, {
+  valid: [
+    {
+      code: `import { Divider } from '@patternfly/react-core'; <Divider />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<Divider />`,
```

Comment:
same comment as on other pr, but this should actually test that your codemod doesn't touch non-pf components, so should have `isVertical` here